ISSUES.md

```diff
@@ -0,0 +1,43 @@
+# **How to create an issue for ONI**
+
+Please help us to improve ONI! Create your issues found following the next steps:
+
+
+## **Create an Issue**
+
+To create an issue please go to [ONI Issues](https://github.com/Open-Network-Insight/open-network-insight/issues), review current open issues for any matches.
+ If there is a match, please feel free to go to the issue and add comments, if you don't find a match please continue creating your issue.
+
+## **Add issue type label**
+
+![Issue type](docs/label-issue-type.bmp)
+
+* **Bug:** Something that is not working properly according the functional spec.
+* **Enhancement:** Add new functionality or improve existing functionality.
+* **Discussion:** Open a discussion related to any topic about ONI solution.
+
+## **Add repo label**
+Once an issue is created, repo tag needs to be added to identify the module that needs to be tracked.
```

Comment:
repo tag needs to be added to identify the module that needs to be tracked -->
add a repo tag to identify the module that needs to be tracked